bin/evm2wasm.js

```diff
@@ -2,16 +2,71 @@
 
 const evm2wasm = require('../index.js')
 const ethUtil = require('ethereumjs-util')
+const async = require('async')
+const argv = require('minimist')(process.argv.slice(2))
```

Comment:
Don't have any preference, but is this better or worse than `yargs` (which is used in many other ethereumjs components)?

Reply:
Looks to be exactly the same functionality.  I was only using minimist because it is already used by a number of ethereumjs libraries (as well as yargs apparently?)